public/apps/configuration/panels/auth-view/auth-view.tsx

```diff
@@ -20,6 +20,7 @@ import { AuthorizationPanel } from './authorization-panel';
 import { API_ENDPOINT_SECURITYCONFIG } from '../../constants';
 import { AppDependencies } from '../../../types';
 import { ExternalLinkButton } from '../../utils/display-utils';
+import './_index.scss';
```

Comment:
It should work without importing `./_index.scss`. Will you please remove this line and see if it still works.

Reply:
tried and it still works. Simplified the code!